src/kv.hpp

```diff
@@ -0,0 +1,16 @@
+#include <optional>
+#include <string>
+
+using Key = std::string;
```

Comment:
Just curious, for the KV db, both key and value are set as strings.

Reply:
We are going to implement an LSM Tree which is built on top of SSTable, right? SSTable just treats values as opaque type. In practice it's acceptable to let the application handle data types with zero cost - you just cast the data to the type you previously stored in the db

Reply:
Gotcha. Thanks for the explanation!